### PR TITLE
chore(deps): update app-runtime to latest

### PR DESCRIPTION
**Pull request by ismay**
https://jira.dhis2.org/browse/LIBS-200. Updating to the new app-runtime release.

Still todo: 

- [ ] Update analytics once this has been published: https://github.com/dhis2/analytics/pull/1000. Otherwise the versions of ui won't match

### CODE REVIEW

File: packages/plugin/src/__tests__/ChartPlugin.spec.js [hunk 1]
```diff
@@ -161,15 +154,6 @@ describe('ChartPlugin', () => {
...
         })
 
-        describe('Year-on-year chart', () => {
-            beforeEach(() => {
-                props.visualization = {
-                    ...yearOverYearCurrentMock,
-                    option1: 'def',
-                }
-            })
-        })
```
Comment (ismay) [inline packages/plugin/src/__tests__/ChartPlugin.spec.js:171]: I've removed this as there is no test in the describe block and it was failing with the new jest version.

**Review by martinkrulltott — APPROVED** [review]

**Comment by martinkrulltott** [comment]
LGTM. Potentially dedupe yarn.lock with `npx yarn-deduplicate yarn.lock` before merging

**Comment by ismay** [comment]
> LGTM. Potentially dedupe yarn.lock with `npx yarn-deduplicate yarn.lock` before merging

Yeah I'll do that. I'll update it with v8 of cli-app-scripts once that's out, and then if 2.37 has been branched it should be good to merge.

**Comment by martinkrulltott** [comment]
Sounds good. 2.37 has been branched to [37.x](https://github.com/dhis2/data-visualizer-app/tree/37.x), so this can actually be merged whenever its ready.

**Pull request closed without merging**